### ISSUE TITLE
Can you add pause feature

### ISSUE THREAD

**Issue by danlniel** [shogo4405/lf.swift#111]

**Comment by shogo4405** [comment]
Yes. But I want to more info "pause".

For example,
1.  NetConneciton is connecting. NetStream is publishing. Audio is mute. Video is black.
1.  NetConneciton is connecting. NetStream is publishing. Audio is mute. Video is last frame.
1.  NetConnection is connecting. NetStream is stop. 

What do you want?

**Comment by danlniel** [comment]
NetConneciton is connecting. NetStream is publishing. Audio is mute. Video is last frame.

**Comment by pzs7602** [comment]
NetConneciton is connecting. NetStream is publishing. Audio is mute. Video is last frame
and an information text displayed on the last frame image, such as "The broadcast is paused ...."

**Comment by shogo4405** [comment]
ok!

**Comment by shogo4405** [comment]
I commited a draft version. new api are here.
```swift
rtmpStream.pause() // if you want to pause a stream
rtmpStream.resume() // if you want to resume a stream
rtmpStream.togglePause() // if you want to paused a stream to resumue or resumed a stream to pause.
```

**Comment by danlniel** [comment]
woah awesome dude, thank you so much

**Comment by San-Jeevan** [comment]
I am really surprised at the agile developement here. well done.

**Comment by shogo4405** [comment]
Released 0.5.4.
